core/src/main/java/GameObjects/GameSettings.java

```diff
@@ -0,0 +1,56 @@
+package GameObjects;
+
+import Scenes.Lobby;
+
+/**
+ * The GameSettings class will store the information for the preferences that were selected in the
+ * lobby screen and will have methods that return values corresponding to the values needed in the
+ * buildUrl method of the UrlBuilderImp class.
+ */
+public class GameSettings {
```

Comment:
Add unit tests for this class.  Especially now that most of the logic in obtaining the API representation of each of the settings is implemented in this class.